src/_data/certifiedNodes.json

```diff
@@ -0,0 +1,55 @@
+[{
```

Comment:
Next time let's construct this from the remote repository kind of like how we include the docs, or the blueprints